### ISSUE TITLE
Section missing from Understanding Conformance

### ISSUE THREAD

**Issue by alastc** [w3c/wcag#807]
In the WCAG 2.0 version of understanding conformance there is a section called "Technical Definition of "Accessibility Support":
https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance#uc-accessibility-supported-definition-head

That is missing in the WCAG 2.1 version:
https://www.w3.org/WAI/WCAG21/Understanding/conformance#accessibility-supported-definition

Looks like an include isn't working.
